GameToken.cs

```diff
@@ -73,6 +73,31 @@ public Int32 MyBalance()
                 }
             } else Error.Throw("Operation denied");
         }
+
+        // Check if GameToken program was called from another program
+        private bool IsCalledFrom(Bytes address) {
+            if(Info.Callers().Length < 2) return false;
+            if(Info.Callers()[Info.Callers().Length-2] != address) return false;
+            return true;
```

Comment:
I think returning the `false` value by default is more secure.  We should return `true` only when we definitely know that this method was called by the `address`. In all other cases, we should return false.